distribution/lib/Standard/Database/0.0.0-dev/src/Internal/SQLite/SQLite_Dialect.enso

```diff
@@ -94,17 +94,8 @@ type SQLite_Dialect
        Prepares a join operation, returning a new table instance encapsulating a
        proper query.
     prepare_join : Connection -> SQL_Join_Kind -> Text -> From_Spec -> From_Spec -> Vector -> Vector -> Vector -> Table
-    prepare_join self connection join_kind new_table_name left_subquery right_subquery on_expressions where_expressions columns_to_select = case join_kind of
-        SQL_Join_Kind.Right ->
-            # We just do a left join with swapped order of sub-queries, while keeping the original order of columns.
-            Database_Join_Helper.default_prepare_join connection SQL_Join_Kind.Left new_table_name right_subquery left_subquery on_expressions where_expressions columns_to_select
-        SQL_Join_Kind.Full ->
-            ## TODO workaround for full joins by left outer + right-anti union
-               https://www.pivotaltracker.com/story/show/184090548
-            Error.throw (Unsupported_Database_Operation.Error "Full outer joins are not YET supported by the SQLite backend. You may need to materialize the Table to perform this operation.")
-        _ ->
-            # Other kinds of joins just fall back to the default logic.
-            Database_Join_Helper.default_prepare_join connection join_kind new_table_name left_subquery right_subquery on_expressions where_expressions columns_to_select
+    prepare_join self connection join_kind new_table_name left_subquery right_subquery on_expressions where_expressions columns_to_select =
+        Database_Join_Helper.default_prepare_join connection join_kind new_table_name left_subquery right_subquery on_expressions where_expressions columns_to_select
```

Comment:
At this point, every call to `prepare_join` just delegates to `Database_Join_Helper.default_prepare_join`.

So I'm wondering if we should just remove this piece of logic.

On one hand, I'd keep it in case in the future we get some other backend that will require some bizarre join workarounds.

On the other hand, it is unlikely + if we encounter such a backend, the workaround needed may not 'fit this mold' anyway. So I'd probably remove it.

Reply:
Simplified the code.